### PR TITLE
sizes overview

### PR DESCRIPTION
**Pull request by marcushohlbein**

### CODE REVIEW

File: client/src/components/ProductDetail/ProductDetail.js [hunk 1]
```diff
@@ -43,9 +43,9 @@ export default function ProductDetail() {
         {data?.shops.map((shop, i) => (
           <ShopListItem
             price={shop.salePrice ?? shop.regularPrice}
-            shipping={shop.shipping}
+            sizes_eu={shop.sizes_eu}
             shopName={shop.shopName}
-            shopUrl={shop.productLink}
+            productLink={shop.productLink}
```
Comment (AnnekeSchuette) [inline client/src/components/ProductDetail/ProductDetail.js]: Might be a monk-ish opinion, but if the productLink is an external link, you could name it respectively, i.e. "productLinkExternal" or "productLinkReseller". In this way it cannot be confused with internal links to a product.

File: client/src/components/ShopListItem/ShopListItem.js [hunk 2]
```diff
@@ -2,23 +2,31 @@ import React from 'react'
 import styled from 'styled-components/macro'
 import Button from '../Button/Button'
 import PropTypes from 'prop-types'
+import formatedSize from '../../lib/formatedSize'
```
Comment (AnnekeSchuette) [inline client/src/components/ShopListItem/ShopListItem.js]: You should fix this little naming typo: "formatedSize" => "format**t**edSize" 🤓✌️

File: client/src/components/ShopListItem/ShopListItem.js [hunk 3]
```diff
@@ -2,23 +2,31 @@ import React from 'react'
...
 
 export default function ShopListItem({
   price,
-  shipping = 'versandkostenfrei',
   shopName,
-  shopUrl,
+  sizes_eu,
+  productLink,
 }) {
+  const imgUrl = './assets/' + shopName + '.svg'
```
Comment (AnnekeSchuette) [inline client/src/components/ShopListItem/ShopListItem.js:13]: Nice solution to get the shop logos dynamically 🎉

File: client/src/components/ShopListItem/ShopListItem.js [hunk 4]
```diff
@@ -28,6 +36,7 @@ ShopListItem.propTypes = {
   shipping: PropTypes.string,
   shopName: PropTypes.string,
   shopUrl: PropTypes.string,
+  sizes_eu: PropTypes.array,
 }
```
Comment (AnnekeSchuette) [inline client/src/components/ShopListItem/ShopListItem.js:46]: 👍

File: client/src/lib/formatedSize.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export default function formatedSize(size) {
+  return size
+    .replace('1/3', ' ⅓')
+    .replace('2/3', ' ⅔')
+    .replace('.0', '')
+    .replace('.', ',')
+}
```
Comment (AnnekeSchuette) [inline client/src/lib/formatedSize.js]: Neat little helper function! 🙂👌

**Review by AnnekeSchuette — COMMENTED** [review]
Great job! 💣✨ Added just a few suggestions. 👩🏻‍🦰✌🏻

File: client/src/components/ProductDetail/ProductDetail.js (hunk 1 above)
Comment (marcushohlbein) [reply client/src/components/ProductDetail/ProductDetail.js]: good advice

File: client/src/components/ShopListItem/ShopListItem.js (hunk 2 above)
Comment (marcushohlbein) [reply client/src/components/ShopListItem/ShopListItem.js]: how could this happend 🙈

**Pull request merged**